controllers/hetznercluster_controller_test.go

```diff
@@ -778,6 +816,16 @@ var _ = Describe("HetznerCluster validation", func() {
 			Expect(testEnv.Create(ctx, hetznerCluster)).To(Succeed())
 		})
 
+		It("should succeed with capi.syself.com/allow-empty-control-plane-address annotation", func() {
```

Comment:
I think we should have two tests here, which reflect the two changes you made in the webhooks!

Reply:
ok I'll extend the tests

Reply:
I still think that we should have two or three tests here:
a) check that it works for empty regions if you specify the annotation
b) check that it works without load balancer and without controlplaneendpoint
c) check that it works without load balancer and without controlplaneendpoint.host

Reply:
ok I added the test cases

Reply:
Sorry, I think there was a misunderstanding... I was talking about tests of the webhook server, which is what we do in the "HetznerCluster validation" section. I thought the other tests in the other sections (which you touched in your latest commit) were already completed...

Reply:
ok I reverted the last commit and added the new tests :-)

Reply:
nice!